core/components/atoms/chip/__tests__/Chip.test.tsx

```diff
@@ -206,4 +206,25 @@ describe('Chip component', () => {
     fireEvent.click(onClose);
     expect(handleOnClose.mock.calls[0][0]).toBe(nameObject);
   });
+
+  it('renders chip component with label of type string', () => {
+    const { getByTestId } = render(<Chip label="ChipLabel" name="Chip" />);
+    const stringLabel = getByTestId('DesignSystem-GenericChip--Text');
+    expect(stringLabel).toHaveClass('Chip-text');
+  });
+
+  it('renders chip component with label not of type string', () => {
```

Comment:
Please change the description. Instead of writing `renders chip component with label not of type string`, we can write something like `renders chip component with label of type ReactElement`